### PR TITLE
[TIMETABLE-126] Added CRUD for Identity-Access and Lesson-Location

### LINKED ISSUE #126: CRUD for Identity-Access and Lesson-Location
Add entities according to Data Domain Model and base CRUD

### PR DESCRIPTION
**Pull request by JJerome-NM**

### CODE REVIEW

**Review by ArsenOOO7 — APPROVED** [review]

File: timetable-academic-catalog/src/main/resources/db/init.sql [hunk 1]
```diff
@@ -1,11 +1,22 @@
...
 
-grant select, insert, delete, update on all tables in schema academic_catalog_microservice to academic_catalog_role;
\ No newline at end of file
+grant
+select,
+insert
+,
+delete,
+update
+on all tables in schema academic_catalog_microservice to academic_catalog_role;
```
Comment (JJerome-NM) [inline timetable-academic-catalog/src/main/resources/db/init.sql]: Need to format this code

File: timetable-group/src/main/resources/db/init.sql [hunk 2]
```diff
@@ -1,11 +1,22 @@
...
 
-grant select, insert, delete, update on all tables in schema group_microservice to group_role;
\ No newline at end of file
+grant
+select,
+insert
+,
+delete,
+update
+on all tables in schema group_microservice to group_role;
```
Comment (JJerome-NM) [inline timetable-group/src/main/resources/db/init.sql]: +1

File: timetable-timetable/src/main/resources/db/init.sql [hunk 3]
```diff
@@ -1,11 +1,22 @@
...
 
-grant select, insert, delete, update on all tables in schema timetable_microservice to timetable_role;
\ No newline at end of file
+grant
+select,
+insert
+,
+delete,
+update
+on all tables in schema timetable_microservice to timetable_role;
```
Comment (JJerome-NM) [inline timetable-timetable/src/main/resources/db/init.sql]: +1

**Review by JJerome-NM — COMMENTED** [review]
I can't request changes so...

**Pull request merged**